firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -60,6 +60,15 @@ class DictionaryEntryDetailSerializer(serializers.HyperlinkedModelSerializer):
     )
     categories = CategorySerializer(many=True)
     site = SiteSummarySerializer()
+    characters = serializers.SerializerMethodField()
+
+    def get_characters(self, entry):
+        if "⚑" in entry.custom_order:
```

Comment:
I can also return a more explicit message here such as "Entry unavailable in games due to it containing unknown characters" etc.